parallel/runner.go

```diff
@@ -129,11 +125,16 @@ func (r *runner) Run() {
 	r.threadsWaitGroup.Wait()
 }
 
-// The producer notifies that no more tasks will be produced.
+// Done notifies that no more tasks will be produced.
```

Comment:
Done notifies... --> Done is used to notify...

Reply:
done